src/main/kotlin/no/skatteetaten/aurora/boober/feature/AlertsFeature.kt

```diff
@@ -46,14 +47,25 @@ class AlertsFeature : Feature {
         "expr",
         "delay",
         "connection",
+        "connections",
         "severity"
     )
 
+    fun containsDeprecatedConnection(spec: AuroraDeploymentSpec): Boolean {
```

Comment:
Sliter litt med å følge hva som skjer i denne metoden, jeg har et forslag som bruker mer gjenkjennelige metoder for spec.
```kotlin
    fun containsDeprecatedConnection(spec: AuroraDeploymentSpec): Boolean {
        val isAlertsConfigured = spec.getSubKeyValues(featureName).isNotEmpty()
        if (!isAlertsConfigured) return false

        val alertFields = spec.getSubKeys(featureName).keys
        return alertFields.any { it.contains(Regex("connection$")) }
    }
```

Denne fungerer med dagens tester hvertfall. Hva tenker du? 
Kunne vi også ha flyttet denne lenger nedover? Liker best at vi holder det konsistent med hva som er på toppen av en feature. Altså de overridden metodene i rekkefølge i henhold til når det skjer.

Reply:
Metoden blir mye enklere å lese, så jeg tok inn endringen.
Jeg er også enig at det bør være konsistent hva som er på toppen, jeg flyttet denne til bunnen for at det ikke skal gå inn i hjelpemetodene for genereringen.